packages/paste-website/src/pages/components/heading/index.mdx

```diff
@@ -199,6 +210,7 @@ import {Heading} from '@twilio-paste/heading';
 | Prop           | Type                  | Description                                                                        | Default          |
 | -------------- | --------------------- | ---------------------------------------------------------------------------------- | ---------------- |
 | as             | asTags                | 'h1', 'h2', 'h3', 'h4', 'h5', 'h6', 'div', 'label', 'span'                         | null             |
+| marginBottom   | string                | 'space0'                                                                           | null             |
```

Comment:
I think you can put 'oneOf(['space0'])` as type here, and then leave a description about how we have default values but allow users to disable it using the `space0` token.

Reply:
Fair. Updated